bin/MadGraph5_aMCatNLO/cards/production/13TeV/DarkMatter/Fermion_Portal_Dirac/fPort_dir_lu1_phi1000_chi400/fPort_dir_lu1_phi1000_chi400_proc_card.dat

```diff
@@ -0,0 +1,44 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.4.3                 2016-08-01         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
```

Comment:
Yes it is normal.
Our model does not deal with lepton. And also the defination is default for madgraph. when you import the model, it will be defined as following,

Defined multiparticle all = g a ve vm vt ve~ vm~ vt~ ur cr t2 dr sr b2 ur~ cr~ t2~ dr~ sr~ b2~ u c t d s b u~ c~ t~ d~ s~ b~ z w+ h w- e- mu- ta- chi e+ mu+ ta+ chi~